diff --git a/crypto_fbe/init.recovery.qcom_decrypt.fbe.rc b/crypto_fbe/init.recovery.qcom_decrypt.fbe.rc
index 95f35c8..72ed77e 100644
--- a/crypto_fbe/init.recovery.qcom_decrypt.fbe.rc
+++ b/crypto_fbe/init.recovery.qcom_decrypt.fbe.rc
@@ -43,17 +43,3 @@ service gatekeeper-1-0-qti /system/bin/android.hardware.gatekeeper@1.0-service-q
     setenv LD_LIBRARY_PATH /vendor/lib64:/vendor/lib:/system/lib64:/system/lib:/sbin
     disabled
     seclabel u:r:recovery:s0
-
-service keystore_auth /system/bin/keystore_auth
-    oneshot
-    user system
-    group root
-    disabled
-    seclabel u:r:recovery:s0
-
-# keystore is started and stopped on demand by TWRP
-service keystore /system/bin/keystore /tmp/misc/keystore
-    user root
-    group root drmrpc readproc log
-    disabled
-    seclabel u:r:recovery:s0
